main.cpp

```diff
@@ -50,87 +50,95 @@ arm.flex();                             // 3)
 //  action 3:   the arm rotates conter-clockwise
 arm.rotateCounterClockwise();           // 4) demonstrates CamelCase
 
-
 //  1)
-//  Noun:
-//  action 1:
-
-//  action 2:
-
-//  action 3:
+//  Noun:       tree
+//  action 1:   the tree gorws
+tree.grow();
+//  action 2:   the tree is getting crooked
+tree.getCrooked();
+//  action 3:   the tree falls
+tree.fall();
 
 //  2)
-//  Noun:
-//  action 1:
-
-//  action 2:
-
-//  action 3:
+//  Noun:       wind
+//  action 1:   the wind blows
+wind.blow();
+//  action 2:   the wind changes direction
+wind.changeDirection();
+//  action 3:   the wind blows stronger
+wind.blowStronger();
 
 //  3)
-//  Noun:
-//  action 1:
-
-//  action 2:
-
-//  action 3:
+//  Noun:       sun
+//  action 1:   the sun rises
+sun.rise();
+//  action 2:   the sun shines
+sun.shine();
+//  action 3:   the sun sets
+sun.set();
 
 //  4)
-//  Noun:
-//  action 1:
-
-//  action 2:
-
-//  action 3:
+//  Noun:       dryer
+//  action 1:   the dryer heats
+dryer.heat();
+//  action 2:   the dryer tumbles
+dryer.tumble();
+//  action 3:   the dryer cools down
+dryer.coolDown();
 
 //  5)
-//  Noun:
-//  action 1:
-
-//  action 2:
-
-//  action 3:
+//  Noun:       hair
+//  action 1:   the hair curls
+hair.curl();
+//  action 2:   the hair changes color
+hair.changeColor();
+//  action 3:   the hair gets tangled
+hair.getTangled();
```

Comment:
As explained in the instructions, here is the obligatory request for some changes to get you familiar with the Pull Request Review process.
REPLACE the highlighted noun/action as follows:
- [ ] come up with a new noun and 3 actions
- [ ] pick a noun that needs `camelCase` for the name
- [ ] pick actions that need `camelCase` for their name.

Do not create an 11th noun.

Once you have completed the above-requested changes, please do the following:
- click each checkbox to mark each requested change as completed.
- click the `[Resolve Conversation]` button.
- Re-request a review of your pull request. It's on the right side of the Pull request page where you originally requested my review.
- Send me the link to your pull request in a new message.